QKSMS/build.gradle

```diff
@@ -100,15 +101,14 @@ dependencies {
     compile 'com.koushikdutta.ion:ion:1.1.7'
     compile 'org.ligi:snackengage:0.4'
     compile 'frankiesardo:icepick:3.2.0'
+    compile 'de.greenrobot:eventbus:2.4.0'
```

Comment:
made the changes.
changed the version and removed eventbus from gradle.

On Sun, May 29, 2016 at 5:18 AM, Marko Stanković <notifications@github.com>
wrote:

> In QKSMS/build.gradle
> <https://github.com/qklabs/qksms/pull/506#discussion_r64995812>:
>
> > @@ -100,15 +101,14 @@ dependencies {
> >      compile 'com.koushikdutta.ion:ion:1.1.7'
> >      compile 'org.ligi:snackengage:0.4'
> >      compile 'frankiesardo:icepick:3.2.0'
> > +    compile 'de.greenrobot:eventbus:2.4.0'
>
> Where's this used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qklabs/qksms/pull/506/files/e6cef0297bd13ed926ce0d7972ae7cd11d175e74..6e09d7a76ed9ff2ac6673c90b79a030ea27d5ab9#r64995812>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG8cjPSbr3cn8b37Nuws9rHW-pvUw8KTks5qGNRKgaJpZM4IpCdE>
> .
>